examples/jsm/renderers/webgl/nodes/GLSLNodeBuilder.js

```diff
@@ -594,6 +594,20 @@ ${ flowData.code }
 
 	}
 
+	getBatchingIndex() {
```

Comment:
I think we could rename this to `getDrawIndex()` and return `null` instead `false`?